classes/feature_overrides.bbclass

```diff
@@ -0,0 +1,18 @@
+def unique_everseen(iterable):
+    "List unique elements, preserving order. Remember all elements ever seen."
+    # unique_everseen('AAAABBBCCDAABBB') --> A B C D
+    # unique_everseen('ABBCcAD', str.lower) --> A B C D
+    import itertools
+    seen = set()
+    seen_add = seen.add
+    for element in itertools.filterfalse(seen.__contains__, iterable):
+        seen_add(element)
+        yield element
+
+python add_feature_overrides () {
+    feature_overrides = filter(None, d.getVar('FEATUREOVERRIDES').split(':'))
+    if feature_overrides:
+        d.prependVar('OVERRIDES', ':'.join(unique_everseen(feature_overrides)) + ':')
+}
+add_feature_overrides[eventmask] = "bb.event.ConfigParsed"
+addhandler add_feature_overrides
```

Comment:
Do we still need the class here now that we have it in meta-mentor?

Reply:
If the other gets merged, then no, we could drop it of course.

Reply:
@kergoth I have merged the PR for class in meta-mentor-common. So kindly update this PR accordingly.

Reply:
@Noor-Ahsan done.